components/Layout.js

```diff
@@ -44,6 +45,12 @@ const Layout = ({children}) => {
                       <Typography variant='body1' color='textPrimary'></Typography>
                     </Grid>
                   </Link>
+                  <hr/>
+                  <a href='https://storage.googleapis.com/buildings-api/LOTB%20-IWWH-Buildings.csv' style={{textDecoration: 'none', color: 'inherit'}}>
+                    <Grid container direction='row'>
```

Comment:
Might need to use IconButton, fine solution for now.

Reply:
Fix:Added_IconButton